### PR TITLE
Restyle + Component Refactor

### PR DESCRIPTION
**Pull request by jackcarlisle**
# ready for review

- [x] restyles the existing views to the design spec
- [x] refactors views into reusable components

#46 
#49

### CODE REVIEW

File: web/controllers/user_controller.ex [hunk 1]
```diff
@@ -12,7 +12,7 @@ defmodule Karma.UserController do
 
   def new(conn, _params) do
     changeset = User.changeset(%User{})
-    render(conn, "new.html", changeset: changeset)
+    render(conn, "new.html", layout: {LayoutView, "login.html"}, changeset: changeset)
```
Comment (JMurphyWeb) [inline web/controllers/user_controller.ex]: can we use a more generic layout name than login.html since it is used by a few different views

Comment (jackcarlisle) [inline web/controllers/user_controller.ex]: Yeah will change now

File: web/templates/layout/app.html.eex [hunk 2]
```diff
@@ -19,7 +19,9 @@
         <nav role="navigation">
           <ul class="nav nav-pills pull-left">
             <%= if @current_user do %>
-            <%= link "Home", to: dashboard_path(@conn, :index), class: "back-button link pa3" %>
+            <a href="/">
+              <img src="/images/logo-blue.png" class="w3"  />
```
Comment (JMurphyWeb) [inline web/templates/layout/app.html.eex]: niiiice

File: web/templates/layout/app.html.eex [hunk 3]
```diff
@@ -19,7 +19,9 @@
         <nav role="navigation">
           <ul class="nav nav-pills pull-left">
             <%= if @current_user do %>
-            <%= link "Home", to: dashboard_path(@conn, :index), class: "back-button link pa3" %>
+            <a href="/">
```
Comment (JMurphyWeb) [inline web/templates/layout/app.html.eex]: we should use path helpers where possible?

**Pull request merged**